packages/react/src/components/UIShell/Content.ts

```diff
@@ -7,15 +7,20 @@
 
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { HTMLAttributes, ReactNode } from 'react';
 import { usePrefix } from '../../internal/usePrefix';
 
+export type ContentProps = HTMLAttributes<HTMLElement> & {
+  children: ReactNode;
+  tagName?: string;
+};
+
```

Comment:
I know `className` is part of `HTMLAttributes`, but it'd be nice to have it explicitly specified here like `children`. Also all these are optional IIRC.

```suggestion
export type ContentProps = HTMLAttributes<HTMLElement> & {
  children?: ReactNode;
  className?: string 
  tagName?: string;
};

```